README.markdown

```diff
@@ -972,6 +974,40 @@ if let number1 = number1 {
 
 Guard statements are required to exit in some way. Generally, this should be simple one line statement such as `return`, `throw`, `break`, `continue`, and `fatalError()`. Large code blocks should be avoided. If cleanup code is required for multiple exit points, consider using a `defer` block to avoid cleanup code duplication.
 
+### Defensive Techniques
+
+Use `sweetAssertionFailure` to catch errors / issues as early as possible.
+**Preferred**:
+```swift
+guard let number1 = number1 else {
+    sweetAssertionFailure("Number should not be nil")
```

Comment:
We still need to `return` after `sweetAssertionFailure` here

Reply:
Should be good now